tests/dummy/app/pods/integration-list/route.js

```diff
@@ -0,0 +1,16 @@
+import Ember from 'ember'
+
+export default Ember.Route.extend({
```

Comment:
We'll need to add some queryParam options here (like `replace`) - this can come from the mixin